app/views/households/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>Household</h1>
+<br>
+<h4>Please create or join a household</h4>
```

Comment:
It seems like this is only the form to create new household, we may need a separate join.html.erb where they can join an existing household.